Resources/Prototypes/_Andromeda/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```
@@ -0,0 +1,96 @@
+      - LightRifleHeavy
+  - type: BallisticAmmoProvider
+    proto: null
+    capacity: 100
+  - type: Item
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Ammunition/Magazine/LightRifle/light_rifle_box.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-1
+      map: ["enum.GunVisualLayers.Mag"]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)